recipe/meta.yaml

```diff
@@ -26,23 +26,20 @@ requirements:
 outputs:
   - name: gymnasium
     build:
-      script: {{ PYTHON }} -m pip install . -vv --no-deps
+      script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
     requirements:
       host:
         - python
-        - cloudpickle
-        - numpy 1.19   # [py<310]
-        - numpy 1.21   # [py==310]
-        - numpy 1.23   # [py>=311]
         - pip
         - setuptools
         - wheel
       run:
         - python
         - cloudpickle >=1.2.0
-        - gymnasium-notices
         - importlib_metadata >=4.8.1  # [py<=39]
-        - {{ pin_compatible('numpy') }}
+        - numpy >=1.21.0
+        - typing-extensions >=4.3.0
+        - farama-notifications >=0.0.1
```

Comment:
It's missing jax-jumpy
https://github.com/Farama-Foundation/Gymnasium/blob/v0.28.1/pyproject.toml#L29

Reply:
Added